native/acter/src/api/pins.rs

```diff
@@ -232,8 +232,8 @@ impl Pin {
         self.content.url.clone()
     }
 
-    pub fn color(&self) -> Option<Color> {
-        self.content.display.as_ref().and_then(|t| t.color.clone())
+    pub fn color(&self) -> Option<u32> {
```

Comment:
Yes it's been there since long but didn't happen to ask on where this would be applied in Pins ?

Reply:
It's a common trait of notes apps to over to change the (background) color of the "post it" - most implement it even before any type of sync feature. The previous Effektio code base and tasks data model already anticipated that to be something people potentially want to customize their objects ... though we never implemented UIs for it.